### PR TITLE
Sigdat 2022 v1.0 Update SIGDAT Officers, Past Officers and news items

### PR DESCRIPTION
**Pull request by ptlord**
Hi Nitin,

Please take a look of the changes. Let me know if you need we need to make further revisions.

Best,

CY

### CODE REVIEW

File: _pages/videos.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ title: "Videos of EMNLP Talks"
 layout: single
 permalink: /videos
 videos:
+- url: https://vimeo.com/search?q=EMNLP+2019
```
Comment (desilinguist) [inline _pages/videos.md:6]:
```suggestion
  - url: https://vimeo.com/search?q=EMNLP+2019
```

File: _pages/home.md [hunk 2]
```diff
@@ -12,11 +12,13 @@ excerpt: "Special Interest Group on Linguistic Data &amp; Corpus-based Approache
 SIGDAT is the [Association for Computational Linguistics](https://aclweb.org) special interest group for linguistic data and corpus-based approaches to natural language processing. SIGDAT organizes the EMNLP Conference. Read more about it [here](/about).
 
 <h2>News</h2>
-
-**November 7, 2019**. SIGDAT president Dr. Jian Su gave [an update](/assets/documents/SIGDAT-2019-update.pdf) on SIGDAT during the closing session of EMNLP-IJCNLP 2019.
 {: .notice--info}
 
-**November 25, 2018**. 
+**December 21, 2021**. Prof. Isabelle Augenstein(http://isabelleaugenstein.github.io/) has been elected as the VP-Elect of SIGDAT 2022.
```
Comment (desilinguist) [inline _pages/home.md:17]:
```suggestion
**December 21, 2021**. Prof. Isabelle Augenstein(http://isabelleaugenstein.github.io/) has been elected as the VP-Elect of SIGDAT 2022. {: .notice--info}
```

File: _pages/home.md [hunk 3]
```diff
@@ -12,11 +12,13 @@ excerpt: "Special Interest Group on Linguistic Data &amp; Corpus-based Approache
...
 
 <h2>News</h2>
-
-**November 7, 2019**. SIGDAT president Dr. Jian Su gave [an update](/assets/documents/SIGDAT-2019-update.pdf) on SIGDAT during the closing session of EMNLP-IJCNLP 2019.
 {: .notice--info}
 
-**November 25, 2018**. 
+**December 21, 2021**. Prof. Isabelle Augenstein(http://isabelleaugenstein.github.io/) has been elected as the VP-Elect of SIGDAT 2022. 
+
+November 7, 2019. SIGDAT president Dr. Jian Su gave [an update](/assets/documents/SIGDAT-2019-update.pdf) on SIGDAT during the closing session of EMNLP-IJCNLP 2019.
```
Comment (desilinguist) [inline _pages/home.md:19]:
```suggestion
November 7, 2019. SIGDAT president Dr. Jian Su gave [an update](/assets/documents/SIGDAT-2019-update.pdf) on SIGDAT during the closing session of EMNLP-IJCNLP 2019. {: .notice}
```

File: _pages/organization.md [hunk 4]
```diff
@@ -11,12 +11,12 @@ toc_icon: "cog"
 
 SIGDAT is run by consensus vote of its organizing committee, consisting of the original SIGDAT organizers and subsequent SIGDAT conference chairs and co-chairs.  SIGDAT has established a Conference Board to help organize the selection of the locations of upcoming EMNLP conferences. 
 
-## Officers in 2019:  
+## Officers in 2022:
```
Comment (desilinguist) [inline _pages/organization.md:14]:
```suggestion
## Officers in 2022
```

**Review by desilinguist — CHANGES_REQUESTED** [review]
Found a few formatting bugs. Please see suggestions.

**Comment by desilinguist** [comment]
Also, in the future, you should be able to submit PRs directly to the original SIGDAT repository without needing to fork. I have added you as a collaborator there with write permissions.

**Comment by ptlord** [comment]
Thank you @desilinguist

**Comment by ptlord** [comment]
Hi Nitin,

The icon for EMNLP 2019 video link is not ready yet, right?
I wonder do we also have video archive for EMNLP 2020 and EMNLP 2021?

Best,

CY

**Comment by desilinguist** [comment]
The 2019 link already works. I just created the channel.

No idea about 2020 and 2021.
________________________________

**Pull request merged**